### PR TITLE
suggested changes to model

### PR DESCRIPTION
**Pull request by gvelez17**
Trying to modify the models to be more in line with the published credentials at http://cooperation.org/credentials/v1/

The models in this app are just a view generated from the actual signed claims, so we can make them for convenience

For exapmle the 'sourceURL' is just a convenience generated from the SourceClaim model, to be used to link in the graph to 'subject' of other claims

### CODE REVIEW

**Comment by gvelez17** [comment]
Goal is that we can have claim as a first class object in the graph

And we can have edges:

  subject -> (has) claim.  (we create a url for the claim when it is written to composedb, it gets a cid)
  subject -> (claim string) object. (node with url of object)
  claim -> (source) sourceURL (node with url of sourceURL)
  claim -> (issuer) issuer (node with url of issuer id)
  
  That would be the goal, to be able to support these types of edges.

File: prisma/schema.prisma [hunk 1]
```diff
@@ -30,33 +30,39 @@ model Claim {
...
+  statement     String?
   aspect        String?
-  howKnown      String?
-  source        String?
+  sourceUrl     String?
   effectiveDate DateTime?
   confidence    Float?
-  reviewRating  Int?
-  createdByUser User          @relation(fields: [userId], references: [id])
-  userId        Int
```
Comment (ArnobChowdhury) [inline prisma/schema.prisma:41]: `userId` is needed for the `createdByUser`. It is reference to the relation it has with the `User` model. You can see `userId` is referenced in `relation(fields: [userId], references: [id])`. You can change the name of the field `userId`. But, a field that will contain `User` model's id is needed. That's how Prisma works.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -30,33 +30,39 @@ model Claim {
...
   lastUpdatedAt DateTime      @default(now())
+  createdByUser User          @relation(fields: [userId], references: [id])
+  ClaimSource   ClaimSource?
   SignedClaim   SignedClaim?
 }
 
 model SignedClaim {
   id              Int             @id @default(autoincrement())
   claim           Claim           @relation(fields: [claimId], references: [id])
-  claimId         Int             @unique
```
Comment (ArnobChowdhury) [inline prisma/schema.prisma:52]: Same issue here. `claimId` is needed as it is referenced in `claim` field.

**Review by ArnobChowdhury — COMMENTED** [review]
You need the `Prisma` VS code extension to work with `Prisma`. Otherwise, you will have tough time catching the errors.

**Comment by gvelez17** [comment]
closing this pr as we have a different set of suggested changes now

**Pull request closed without merging**